### ISSUE TITLE
Checkbox Cross Color / Cross not showing

### ISSUE THREAD

**Issue by South2AK** [enisn/Xamarin.Forms.InputKit#256]
Hi,

I´m trying to add a Checkbox via Code Behind.

```
Plugin.InputKit.Shared.Controls.CheckBox C = new Plugin.InputKit.Shared.Controls.CheckBox();
                                                C.Text = Txt2;
                                                C.Type = Plugin.InputKit.Shared.Controls.CheckBox.CheckType.Material;
                                                C.IconColor = Color.White;
                                                FrameAbschnittSL.Children.Add(C);

```

![Unbenannt](https://user-images.githubusercontent.com/48831239/170650394-accbc688-c8f8-4ef7-9c2e-aa6b5e4256ae.png)

The other Checkbox Types will just change the border color, but won´t add a Icon / Cross.

Android is being used with dark mode. 
Version 4.0.0

**Comment by enisn** [comment]
Can you share what is your Xamarin.Forms version?

**Comment by enisn** [comment]
I see the problem occurs because of embedded resource namespaces. I've fixed it with #257 and I'll release it asap.

**Comment by South2AK** [comment]
Is working now.

Thank you very much.
